cql3/CqlLexer.g4

```diff
@@ -1,246 +1,1073 @@
+
 lexer grammar CqlLexer;
 
-LR_BRACKET: '(';
-RR_BRACKET: ')';
 
-LC_BRACKET: '{';
-RC_BRACKET: '}';
+LR_BRACKET
+   : '('
```

Comment:
I prefer one-line declaration for tokens (here and further):

```antlr
LR_BRACKET: '(';
```

It's shorter and more clear.